package/CHANGELOG

```diff
@@ -13,10 +13,11 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-  
+
 ??/??/?? IAlibay, pgbarletta, mglagolev, hmacdope, manuel.nuno.melo, chrispfae, 
          ooprathamm, MeetB7, BFedder, v-parmar, MoSchaeffler, jbarnoud, jandom,
-         xhgchen, jaclark5, DrDomenicoMarson
+         xhgchen, jaclark5, DrDomenicoMarson, SophiaRuan
+
```

Comment:
I know this is annoying @SophiaRuan but this should show no diff **except your name**

Reply:
Hi @hmacdope, I just made some changes. Could you help me check a bit? Thank you for your help!